examples/firebase/index.js

```diff
@@ -0,0 +1,179 @@
+                    newPass.headerFields.push({
+                        key: `header${index}`,
+                        label: field.label,
+                        value: field.value,
+                    });
+                }
+                });
+
+            request.body.primary.forEach((field, index) => {
+                if (field.label !== '' || field.value !== '') {
+                    newPass.primaryFields.push({
+                        key: `primary${index}`,
+                        label: field.label,
+                        value: (currentPassType == "boardingPass") ? field.value.toUpperCase() : field.value,
+                    });
+                }
+                });
+
+            request.body.secondary.forEach((field, index) => {
+            if (field.label !== '' || field.value !== '') {
+                newPass.secondaryFields.push({
+                    key: `secondary${index}`,
+                    label: field.label,
+                    value: field.value,
+                    textAlignment: (index === request.body.secondary.length - 2 || index === request.body.secondary.length - 1) ? "PKTextAlignmentRight" : "PKTextAlignmentLeft",
+                });
+            }
+            });
+
+            request.body.auxiliary.forEach((field, index) => {
+                if (field.label !== '' || field.value !== '') {
+                    newPass.auxiliaryFields.push({
+                        key: `auxiliary${index}`,
+                        label: field.label,
+                        value: field.value,
+                        textAlignment: (index === request.body.secondary.length - 2 || index === request.body.secondary.length - 1) ? "PKTextAlignmentRight" : "PKTextAlignmentLeft",
+                    });
+                }
+                });
+
+            if (!request.body.codeAlt || request.body.codeAlt.trim() === "") {
+                newPass.setBarcodes({
+                    message: request.body.qrText,
+                    format: `PKBarcodeFormat${request.body.codeType}`,
+                    messageEncoding: "iso-8859-1",
+                });
+            } else {
+                newPass.setBarcodes({
+                    message: request.body.qrText,
+                    format: `PKBarcodeFormat${request.body.codeType}`,
+                    messageEncoding: "iso-8859-1",
+                    altText: request.body.codeAlt,
+                });
+            }
+            
+            // Downloading thumbnail and logo files from Firebase Storage and adding to pass
+            if (currentPassType == "generic" || currentPassType == "eventTicket") {
+                const thumbnailFile = request.body.thumbnailFile
+                const tempPath1 = path.join(os.tmpdir(), thumbnailFile)
+                try {
+                    await storageRef.file(`thumbnails/${thumbnailFile}`).download({destination: tempPath1})
+                } catch (error) {
+                    //
```

Comment:
Please put at least a `console.error` here to print the error

Reply:
Haha yep sorry, I'll go through and put those in